src/openscm_runner/adapters/ciceroscm_adapter/ciceroscm.py

```diff
@@ -0,0 +1,69 @@
+"""
+CICEROSCM adapter
+"""
+import logging
+import os.path
+from subprocess import check_output  # nosec
+
+from ..base import _Adapter
+from ._run_ciceroscm_parallel import run_ciceroscm_parallel
+
+LOGGER = logging.getLogger(__name__)
+
+
+class CICEROSCM(_Adapter):  # pylint: disable=too-few-public-methods
+    """
+    Adapter for CICEROSCM
+    """
+
+    def __init__(self):  # pylint: disable=useless-super-delegation
+        """
+        Initialise the CICEROSCM adapter
+
+        """
+        super().__init__()
+
+    def _init_model(self):  # pylint: disable=arguments-differ
+        pass
+
+    def _run(self, scenarios, cfgs, output_variables, output_config):
+        """
+        Run the model.
+
+        This method is the internal implementation of the :meth:`run` interface
+
+        cfgs is a list of indices to run
+        """
+        if output_config is not None:
+            raise NotImplementedError("`output_config` not implemented for Cicero-SCM")
+
+        runs = run_ciceroscm_parallel(scenarios, cfgs, output_variables)
+
+        return runs
+
+    @classmethod
+    def get_version(cls):
+        """
+        Get the CICEROSCM version being used by this adapter
+
+        Returns
+        -------
+        str
+            The CICEROSCM version id
+
+        Raises
+        ------
+        OSError
+            The Cicero-SCM binary cannot be run on the operating system
+        """
+        exec_call = os.path.join(
+            os.path.dirname(__file__), "utils_templates", "run_dir", "scm_vCH4fb"
+        )
+        try:
+            check_output(exec_call)
+        except OSError as orig_exc:
+            raise OSError(
+                "Cicero-SCM is not available on your operating system"
+            ) from orig_exc
+
+        return "v2019vCH4"
```

Comment:
I'm guessing this has to behard-coded i.e. you can't tell which version of CICERO-SCM you have just from the binary?

Reply:
You would be right

Reply:
Ok cool, one for another day then (see #49)